Olymp.Nodes/Master/ConfigurationServerNode.cs

```diff
@@ -54,8 +57,12 @@ protected override (Command cmd, IMessage unencryptedMessage) Handle(BaseMessage
 
                     try
                     {
+                        //Check Permission level of issuer
+                        var issuer = UserRepository.Instance.GetUser(message.User);
+                        if (!IssuerIsAdmin(issuer)) return (Command.FAIL, setUserLevelMessage);
```

Comment:
If IssuerIsAdmin is one line, you don't have to create a method for it.  
```suggestion
                        if (issuer.AuthLevel == ADMIN_LEVEL)
                            return (Command.FAIL, setUserLevelMessage);
```

Reply:
@Azer0s If someone tries to elevate their privileges, how do we tell them? They only receive a fail command with no explanation.  
Do we send a SingleValueMessage with the error, or maybe create a new Message for Errors

Reply:
@Mitiko To be quite honest...I don't know. In classic web applications, to avoid exploits, one never tells the user what's wrong. On the other hand...Olymp is not a web application but a compute engine. Hmmm...I'd say we tell the user.

Reply:
If that's a concern, I think we only disclose these couple of lines and they look secure enough to me

Reply:
But do we create a new Message type?